services/zookeeper-utils/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>datawave-microservice-parent</artifactId>
+        <groupId>gov.nsa.datawave.microservice</groupId>
+        <version>1.3</version>
+    </parent>
+    <artifactId>zookeeper-utils</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <properties>
+        <version.curator>2.11.0</version.curator>
```

Comment:
Looks like this version of curator is way out of date. If we're moving things around (and re-testing anyway), it might be worth moving to a more recent version at the same time. I think the only reason that might not be possible is zookeeper version issues. We need to be compatible with whatever zookeeper is running along with Accumulo 1.9.2 and on the primary user systems.